.github/workflows/deploy.yaml

```diff
@@ -398,10 +388,15 @@ jobs:
 
       - name: 'Register participant'
         run: |
-          mvn dependency:copy -Dartifact=org.eclipse.dataspaceconnector.registrationservice:registration-service-cli:1.0.0-SNAPSHOT:jar:all -DoutputDirectory=.
-          java -jar registration-service-cli-1.0.0-SNAPSHOT-all.jar -s=$REGISTRATION_SERVICE_API_URL participants add --request='{ "name": "${{matrix.participant}}", "supportedProtocols": [ "ids-multipart" ], "url": "http://${{ env.EDC_HOST }}:8282" }'
+          mvn dependency:copy -Dartifact=org.eclipse.dataspaceconnector.registrationservice:registration-service-cli:0.0.1-SNAPSHOT:jar:all -DoutputDirectory=.
```

Comment:
see above. not sure why `mvn` is used. using a hard-coded version may also be problematic

Reply:
As above. We'll have to know the version to consume anyway

Reply:
Can we make this an input property that you have to manually enter when triggering the job? Could be in another PR.

Reply:
@paullatzelsperger MVD usages a specific version/checkout of registration service and that's why we can only use a compatible client jar version in CI. That's why it is hard-coded here to for compatibility.

Reply:
Lets get this merged for now. But down the road we'll have to figure out how to better and more robustly download/build/get the `registration-service.jar`

Reply:
added #32